src/Editor/Resources_BACKUP_7056.cpp

```diff
@@ -0,0 +1,261 @@
+#include "Resources.hpp"
```

Comment:
Looks like residue from resolving a merge conflict. This file should not be added to the repo.